projects/helper/tokenMapping.js

```diff
@@ -135,6 +135,7 @@ const fixBalancesTokens = {
     '0xf4c20e5004c6fdcdda920bdd491ba8c98a9c5863': { coingeckoId: 'bouncebit', decimals: 18 },
     '0x77776b40c3d75cb07ce54dea4b2fd1d07f865222': { coingeckoId: 'tether', decimals: 18 },
     '0xF5e11df1ebCf78b6b6D26E04FF19cD786a1e81dC': { coingeckoId: 'bitcoin', decimals: 18 },
+    '0x8f083EaFcbba2e126AD9757639c3A1E25a061A08': { coingeckoId: 'wrapped-bitcoin', decimals: 18 },
```

Comment:
I am sorry, but what is this token, without more info, I cant merge this

Reply:
According to the native stBBTC contract design, directly using stBBTC to create a token pool (e.g., AAVE's Token Pool) can cause precision loss in the calculation of its BB token rewards. To ensure accurate BB reward calculations for stBBTC, we wrap stBBTC again. This wrapping adapts to the BB calculation and distribution in the pool model, similar to wstETH. For the specific contract implementation, please refer to https://bbscan.io/contract/0x8f083EaFcbba2e126AD9757639c3A1E25a061A08.

Reply:
yes, but what is the project using this?

Reply:
Our Lending Protocol LenB, which is about to be launched on BounceBit, will use this contract